### PR TITLE
Add target data

### PR DESCRIPTION
**Pull request by sbidari**
Adds target data from the latest release of NHSN hospital admission data

### CODE REVIEW

File: src/code/get_target_data.r [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Obtain covid target data
+
+health_data <- RSocrata::read.socrata(url = "https://data.cdc.gov/resource/ua7e-t2fy.json") |> # nolint
+  dplyr::filter(weekendingdate >= as.Date("2024-11-02"))
+
+formatted_data <- health_data |>
+  dplyr::select(jurisdiction, weekendingdate, totalconfc19hosppats) |>
+  dplyr::rename(
+    "value" = "totalconfc19hosppats",
```
Comment (dylanhmorris) [inline src/code/get_target_data.r:9]: we want `totalconfc19newadm` (new admissions). But we can also use `pull_nhsn()` from `forecasttools`

Comment (sbidari) [reply src/code/get_target_data.r:9]: I decided to wait until Wednesday data release to update this. I figure we will want the early released data

**Comment by sbidari** [comment]
Ready for re-review @dylanhmorris 
I changed starting date to reflect first full week of mandatory reporting period.
I will add GH action to update the data every week on a separate PR.

File: src/code/get_target_data.r (hunk 1 above)
Comment (sbidari) [reply src/code/get_target_data.r]: changed in https://github.com/CDCgov/covid19-forecast-hub/pull/42/commits/a21383b7de636f4e09460baf5cab2d8685b4f8cd

**Pull request merged**